plugins/tutor-contrib-paragon/tutorparagon/templates/paragon/build/paragon-builder/entrypoint.sh

```diff
@@ -41,6 +41,33 @@ parse_args() {
     printf '%s\n' "$@"
 }
 
+build_css_bundle() {
+    # This function builds a CSS bundle using PostCSS.
+    # It takes the path to the index.css file as an argument.
+    # Usage: build_css_bundle <index_css_file>
+
+    local index_css_file="$1"
+
+    local bundle_directory=$(dirname "$index_css_file")
+    local bundle_name=$(basename "$bundle_directory")
+    local minified_output_file="$bundle_directory/${bundle_name}.min.css"
+
+    if npx postcss "$index_css_file" \
+        --use postcss-import \
+        --use postcss-custom-media \
+        --use postcss-combine-duplicated-selectors \
+        --use postcss-minify \
+        --no-map \
+        --output "$minified_output_file"; then
+        
+        echo "Successfully created bundle: $bundle_name"
+        return 0
+    else
+        echo "Failed to build CSS bundle: $bundle_name" >&2
+        return 1
+    fi
+}
```

Comment:
I'm not opposed to having this live here, but if we think postcss bundling paragon generated css will be a common thing then maybe it makes sense to build that functionality into the Paragon CLI. @PKulkoRaccoonGang thoughts?